### PR TITLE
Update complete-postgres-manifest.yaml

### PR DESCRIPTION
**Pull request by wasap**
add maxDBConnections settings to connectionPooler

### CODE REVIEW

**Comment by FxKu** [comment]
:+1:

File: manifests/complete-postgres-manifest.yaml [hunk 1]
```diff
@@ -157,6 +157,7 @@ spec:
 #    mode: "transaction"
 #    schema: "pooler"
 #    user: "pooler"
+#    maxDBConnections: 300
```
Comment (FxKu) [inline manifests/complete-postgres-manifest.yaml:160]:
```suggestion
#    maxDBConnections: 60
```

**Comment by idanovinda** [comment]
👍

**Comment by FxKu** [comment]
Thank you @wasap for your contribution!

**Pull request merged**